Sources/graphql-swift/graphqlClient.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+open class GraphqlClient {
+    let url: URL!
+    
+    public init(_ endpoint: String) {
+        url = URL(string: endpoint)!
+    }
```

Comment:
May I propose changing the parameter type to `URL` that way whoever is initializing the client is responsible for producing a valid URL. And if that’s user input they can validate it and/or throw and error back to the user.
```suggestion
    public init(endpoint: URL) {
        self.url = url
    }
```